documentation/UserGuide.md

```diff
@@ -159,4 +159,14 @@ After the first launch, the app uses the bsaemap layers that it loaded previousl
 
 ### Simulating messages
 
-You can run the Message Simulator application included in the [Vehicle Commander](https://github.com/Esri/vehicle-commander) template ([source](https://github.com/Esri/vehicle-commander/tree/master/source/MessageSimulator), [sample messages](https://github.com/Esri/vehicle-commander/blob/master/source/MessageSimulator/SampleMessages/AFM_Sample_MSGS-719.xml). This is especially useful for testing and demonstration purposes. Note that these simulated messages will not make it to Squad Leader running on an emulator ([more info](#running-on-an-android-emulator)).
+You can run the [GeoMessage Simulator application](https://github.com/Esri/geomessage-simulator-qt) to send messages to Squad Leader. GeoMessage Simulator is especially useful for testing and demonstration purposes. Note that these simulated messages will not make it to Squad Leader running on an emulator ([more info](#running-on-an-android-emulator)). (TODO add link to compiled GeoMessage Simulator when it is posted)
```

Comment:
I guess we could add that link now that we know where this is: http://www.esri.com/apps/products/download/index.cfm#ArcGIS_for_the_Military (I'll try to remember to get that app name changed to a more useful/obvious "GeoMessage Simulator"